cloudmarker/clouds/azurecloud.py

```diff
@@ -79,20 +79,20 @@ def read(self):
             resource_list = resource_client.resources.list()
 
             # Retrieve data using each iterator.
-            for doc in itertools.chain(
-                    _get_doc(vm_list, 'virtual_machine', subscription_id),
-                    _get_doc(app_gw_list, 'app_gateway', subscription_id),
-                    _get_doc(lb_iter, 'lb', subscription_id),
-                    _get_doc(nic_list, 'nic', subscription_id),
-                    _get_doc(nsg_list, 'nsg', subscription_id),
-                    _get_doc(pubip_list, 'public_ip', subscription_id),
-                    _get_doc(storage_account_list, 'storage_account',
+            for record in itertools.chain(
+                    _get_record(vm_list, 'virtual_machine', subscription_id),
+                    _get_record(app_gw_list, 'app_gateway', subscription_id),
+                    _get_record(lb_iter, 'lb', subscription_id),
+                    _get_record(nic_list, 'nic', subscription_id),
+                    _get_record(nsg_list, 'nsg', subscription_id),
+                    _get_record(pubip_list, 'public_ip', subscription_id),
+                    _get_record(storage_account_list, 'storage_account',
                              subscription_id),
```

Comment:
Make checks fails here and at line `92` for indentation.

Reply:
Fixed [lines 90 and 92](https://github.com/cloudmarker/cloudmarker/pull/74/files#diff-4fb06491b278fdc6193c4b1e9b8d0284R89).